justbuild.it-WebApp/src/main/java/justbuild/it/web/app/service/OfferEditService.java

```diff
@@ -0,0 +1,39 @@
+package justbuild.it.web.app.service;
+
+import justbuild.it.web.app.dto.OfferDto;
+import justbuild.it.web.app.entity.Offer;
+import justbuild.it.web.app.mapper.OfferMapper;
+import justbuild.it.web.app.repository.OfferFileRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static justbuild.it.web.app.entity.constants.AppConstants.OFFERS_FILEPATH;
+
+@Service
+public class OfferEditService {
```

Comment:
1. zmieniłbym nazwę na klasy na OfferEditionService, żeby nazwa pasowała do pozostałych klas serwisowych :) 
2. Mariusz sugerował, żeby klasy serwisowe gdzie jest logika były package-private, dobrze byłoby się tego trzymać - usunąłbym słowo public z nazwy klasy i konstruktora, metody bez zmian - publiczne:
class OfferEditionService
OfferEditionService(OfferFileRepository offerFileRepository, OfferMapper mapper) {
        this.offerFileRepository = offerFileRepository;
        this.mapper = mapper;
    }
    
    W związku ztym nie możemy wstrzykiwac klasy OfferEditionService bezpośrednio do kontrollera, tylko do klasy OfferService, która jest w tym samym pakiecie !!!
    
    3. żeby wszystkie nasze klasy spełniały wymogi SOLID trzeba dodać interfejs:
    @Service
public interface OfferEditionServiceInterface {

    OfferDto getOfferDtoById(Long id);

    void updateOffer(OfferDto editedOfferDto);
}

4. klasa powinna go implementować:

@Service
class OfferEditionService implements OfferEditionServiceInterface {